### PR TITLE
feat: sort translation strings

### PR DESCRIPTION
**Pull request by danielgent**
~If~ this works *SO* let's do it everywhere! :superhero:

### CODE REVIEW

File: src/localesLogin2/en/default.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "Server error: Invalid code": "Invalid code",
```
Comment (danielgent) [inline src/localesLogin2/en/default.json:2]: interesting that this comes first *BUT* we shouldn't have this! :smile:  Why is it capitalized

File: src/localesLogin2/nb/default.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+  "Server error: Invalid code": "Ugyldig kode",
```
Comment (danielgent) [inline src/localesLogin2/nb/default.json:2]: damn it's just outputting random characters!

**Review by frankjonatanschultz — APPROVED** [review]

**Comment by kkortes** [comment]
Can you enlighten me why we need `sort-json`?

They _are_ sorted from i18nexus.

And if they aren't for specific projects we should ask them to fix that for us. Or am I missing something?

**Comment by frankjonatanschultz** [comment]
> @danielgent
> 
> Can you enlighten me why we need `sort-json`?
> 
> They _are_ sorted from i18nexus.
> 
> And if they aren't for specific projects we should ask them to fix that for us. Or am I missing something?

They are random for me in the checkout repo at least. And looking at Dans PRs in login2 they seem to be random there as well https://github.com/sesamyab/login2/pull/608/files

**Review by majkiee — APPROVED** [review]

**Pull request merged**